src/server/tran_server.cpp

```diff
@@ -725,14 +719,27 @@ tran_server::ps_connector::try_connect_to_all_ps (cubthread::entry &)
 	   * TODO It can be too verbose now since it always complain to fail to connect when a PS has been stopped.
 	   * Later on, this job is going to be triggered by a cluster manager or cub_master when a PS is ready to connect.
 	   */
-	  (void) conn->connect ();
+	  if (conn->connect () == NO_ERROR)
+	    {
+	      newly_connected = true;
+	    }
 	}
 
       if (m_terminate.load ())
 	{
 	  break;
```

Comment:
Nit.
Why not `return` in case of `m_terminate`?

Reply:
It was the same, but with this change, it gets different. I'll make it return.